hazelcast/src/test/java/com/hazelcast/util/executor/LoggingScheduledExecutorTest.java

```diff
@@ -75,24 +77,32 @@ public void tearDown() throws Exception {
     }
 
     @Test
+    @Category(SlowTest.class)
     public void no_remaining_task_after_cancel() throws Exception {
         executor = new LoggingScheduledExecutor(logger, 1, factory);
+        ((LoggingScheduledExecutor) executor).enablePeriodicPurge(Executors.newSingleThreadExecutor());
```

Comment:
Shouldn't you terminate this executor after test: `Executors.newSingleThreadExecutor()`